ArcGISRuntimeSDKQt_QMLSamples/Routing/NavigateRoute/main.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <QSettings>
+#include <QGuiApplication>
+#include <QQuickView>
+#include <QCommandLineParser>
+#include <QDir>
+#include <QQmlEngine>
+
+#include "NavigateRouteSpeaker.h"
+
+static QObject *example_qobject_singletontype_provider(QQmlEngine *engine, QJSEngine *scriptEngine)
```

Comment:
Did you forget to rename this method, or even forget to delete it? Can't see where it's used.

Reply:
I forgot to remove it! Thanks for catching that